### ISSUE TITLE
BUG: webhook, missing update of pending status

### ISSUE THREAD

**Issue by Zambiorix** [pretix/pretix#1174]
When an order has been placed and paid.
* create a refund in the order details backend page
* in the "what should happen to the order" select -> "Mark order as pending"
finish the refund

No webhook is called is called to update the status to pending
So, our own backend is not in sync with Pretix

If "Cancel the order" is selected in  "what should happen to the order", the correct webhook is called.

**Comment by raphaelm** [comment]
You are right, there currently is no subscribable event when an order is going to status "pending" by a manual action.

**Comment by Zambiorix** [comment]
So currently there is no way to guarantee an exact order sync with an external system?

Is this an objective for the near future?

**Comment by raphaelm** [comment]
Well, I'm all for adding a proper webhook for this status change, but no, it's not high on our roadmap right now.

"Guaranteed exact order sync" is certainly not an explicit objective or guarantee we ever made before or are planning to make, I can't even imagine how we'd want to ensure that, there's lots of things that can happen to orders. But yes, at least covering all status changes would be great.

**Comment by Zambiorix** [comment]
Well, best effort is good enough for me. With eventual consistency for the backend via Pretix API and scanning through the orders (every x hours or so)
